### PR TITLE
Functional + phpunit tests

### PR DESCRIPTION
**Pull request by ondrajodas**
Jira - https://keboola.atlassian.net/browse/COM-667
https://keboola.atlassian.net/browse/COM-668

### CODE REVIEW

**Comment by ondrajodas** [comment]
@pivnicek jestli si najdeš chvilku 🙏

File: src/Application.php [hunk 1]
```diff
@@ -135,8 +135,15 @@ public function writeFullFromAdapter(array $tableConfig, IAdapter $adapter): voi
 
     private function getManifest(string $tableId): array
     {
+        $tablePath = $this['parameters']['data_dir'] . '/in/tables/' . $tableId . '.csv.manifest';
```
Comment (pivnicek) [inline src/Application.php:138]: to be more clear I would call this `manifestPath` :)

**Review by pivnicek — DISMISSED** [review]
LGTM, tests <3

**Comment by ondrajodas** [comment]
@pivnicek just one more time please :)

**Review by pivnicek — APPROVED** [review]
diky :-)

**Pull request merged**